string date
string time

float32 latitude
float32 longitude
float32 height

string solution
int64 num_satelites

float32 stn
float32 ste
float32 stu
float32 stne
float32 steu
float32 stun

float32 age
float32 ratio
